commons/com.b2international.commons/src/com/b2international/commons/options/Options.java

```diff
@@ -67,9 +67,24 @@
 	 *             if the specified key is <code>null</code>
 	 */
 	Object get(String key);
+	
+	/**
+	 * Returns the value to which the specified Enum key's name is mapped, or {@code null} if this map contains no mapping for the key.
+	 *
+	 * <p>
+	 * More formally, if this map contains a mapping from a key {@code k} to a value {@code v} such that {@code (key==null ? k==null :
+	 * key.equals(k))}, then this method returns {@code v}; otherwise it returns {@code null}. (There can be at most one such mapping.)
```

Comment:
```suggestion
	 * key.name().equals(k))}, then this method returns {@code v}; otherwise it returns {@code null}. (There can be at most one such mapping.)
```